### DISCUSSION TITLE
GitHub.com showing shell as a language but no *.sh files

### DISCUSSION THREAD

**Discussion by thunder-coding** [github-linguist/linguist#5180] [Classification] [answered]
## Problem description 

I am having a TypeScript project, in which I have used no other language than TypeScript. 

The bar shows 99.7% TypeScript and 0.3% Shell. I had never added a `*.sh` file in the repository.

Here's what it looks like
 ![Screenshot_20210201-083642~2.png](https://user-images.githubusercontent.com/68849076/106410356-b0c4dc80-6468-11eb-9c0b-3b6a1847e8c4.png)

### URL of the affected repository:
 
https://github.com/thunder-coding/BotRanger

### Last modified on: 
2021-02-01

### Expected language:
 100% TypeScript
 
### Detected language:
 99.7% TypeScript + 0.3% Shell

**Comment by lildude** [comment, marked as answer]
The `.env.example` file in your repo is a named file associated with Shell:

https://github.com/github/linguist/blob/299318747a903a90581f46e637f7d3be387ff844/lib/linguist/languages.yml#L5180-L5209

**Reply by thunder-coding** [reply]
Thanks for your answer.

Just in case anyone else is wondering how to solve this issue.

```gitattributes
.env.example -linguist-detectable
```

Add the above line to `.gitattributes` file.

For more details visit https://github.com/github/linguist#detectable
